### DISCUSSION TITLE
Error refreshing dataset

### DISCUSSION THREAD

**Discussion by MatthewDiPietroTR** [nbrown02/FlowViz#103] [Q&A]
I've recently started receiving this error when updating the datasource in the web interface. I have absolutely no problems in Power BI Desktop. Has anyone ever encountered something like this or have ideas for how to debug?

> Data source error: Expression.Error: We cannot convert the value <ccon>"[Table]" to type Table.. Value = [Table]. </ccon>. The exception was raised by the IDbCommand interface. Table: WorkItems In Progress2.
> Cluster URI: WABI-WEST-US-D-PRIMARY-redirect.analysis.windows.net
> Activity ID: 00b38f2d-2f52-4d9c-9ddc-af0d13e3e1a8
> Request ID: 461a1ef7-e36d-8ad7-eea0-90387f6ecfb9
> Time: 2023-07-06 17:21:49Z

**Comment by nbrown02** [comment]
Hey Matthew,Not seen this at all on any reports :/ Have you logged with Microsoft for their view?Sent from my iPhoneOn 6 Jul 2023, at 18:32, Matthew DiPietro ***@***.***> wrote:﻿
I've recently started receiving this error when updating the datasource in the web interface. I have absolutely no problems in Power BI Desktop. Has anyone ever encountered something like this or have ideas for how to debug?

Data source error: Expression.Error: We cannot convert the value "[Table]" to type Table.. Value = [Table]. . The exception was raised by the IDbCommand interface. Table: WorkItems In Progress2.
Cluster URI: WABI-WEST-US-D-PRIMARY-redirect.analysis.windows.net
Activity ID: 00b38f2d-2f52-4d9c-9ddc-af0d13e3e1a8
Request ID: 461a1ef7-e36d-8ad7-eea0-90387f6ecfb9
Time: 2023-07-06 17:21:49Z

**Reply by MatthewDiPietroTR** [reply]
I'm not really sure what the problem was, but I narrowed it down to a modification I had to make to remove duplicate records due to multiple backlogs looking at the same area paths and duplicating records due to multiple board locations.

**Comment by nbrown02** [comment]
Ah ok - difficult for me to reproduce that one then! Let me know if there is anything needs changing to the template :)Sent from my iPhoneOn 7 Jul 2023, at 20:03, Matthew DiPietro ***@***.***> wrote:﻿
I'm not really sure what the problem was, but I narrowed it down to a modification I had to make to remove duplicate records due to multiple backlogs looking at the same area paths and duplicating records due to multiple board locations.
